cookbook/form/create_form_type_extension.rst

```diff
@@ -319,3 +311,15 @@ next to the file field. For example::
 
 When displaying the form, if the underlying model has already been associated
 with an image, you will see it displayed next to the file input.
+
+Generic Form Type Extensions
+----------------------------
```

Comment:
That's fine with me :). In the paragraph below, we should mention the 'getExtendedType' function specifically so they know where they'd exactly put 'text' or 'form' (maybe even show a small code block)